elasticdl/python/tests/layer_test.py

```diff
@@ -373,6 +373,15 @@ def test_embedding_layer_gradient_with_sparse_inputs(self):
                         ).all()
                     )
 
+    def test_embedding_layer_with_none_shape_input(self):
+        output_dim = 8
+        embedding_size = 16
+        layer = create_embedding_layer(embedding_size, output_dim)
+        input = tf.keras.layers.Input(shape=(None,))
+        embedding_output = layer(input)
+        embedding_output_shape = embedding_output.get_shape().as_list()
+        assert embedding_output_shape == [None, 1, 8]
```

Comment:
You can also use self.assertEquals for consistency with other tests

Reply:
OK, do it.